scripts/seed-database/container.sh

```diff
@@ -32,21 +29,19 @@ for table in "${tables[@]}"; do
   fi
 done
 
-if [[ ${RESET} == "reset_flows" ]]; then
-  psql ${LOCAL_PG} --command="TRUNCATE TABLE flows CASCADE;"
-fi
+psql --quiet ${REMOTE_PG} --command="\\copy (SELECT DISTINCT ON (flow_id) id, data, flow_id, summary, publisher_id, created_at FROM published_flows ORDER BY flow_id, created_at DESC) TO '/tmp/published_flows.csv' (FORMAT csv, DELIMITER ';');"
+echo published_flows downloaded
```

Comment:
are these lines still necessary? published_flows should be in concatened command below, and I _think_ fullly covered by write/published_flows.sql now?

Reply:
resolved this in slack!